### ISSUE TITLE
[BUG] Unable to check minecraft server 

### ISSUE THREAD

**Issue by itsmemac** [phpservermon/phpservermon#1031]
Minecraft server ping returns offline - Maybe this script is not intended to check that, but gave it a try

Failed to connect to the following server:
Server: Minecraft server
IP: XXX.XXX.XXX.XXX (Masked the ip)
Port: 25510
Error: Connection refused

Unable to check minecraft server uptime

 - Version - 3.5.2
 - PHP - 7.2

**Comment by nightw0lv** [comment]
connection refused says it all... the connection was refused by whom can this be done? only the minecraft server can refuse that.

**Comment by BetTD** [comment]
I'm unable to reproduce this issue on my end.

A service check really just checks if a specific port is open and listening on the specified host (I know there's more to this). If the connection is being refused, that means the Minecraft server is probably turned off or is listening on another port (e.g. 25565). This may also be the result of a firewall rule that is actively refusing the connection instead of dropping it.

In conclusion, this is most likely not a bug with PSM.

**Comment by itsmemac** [comment]
I get that, but other apps are able to ping the server and get response. Only this was not able to ping and get response 🤔

Ports are open and firewall is configured 
It's an active minecraft server with query enabled

**Comment by wienfuchs** [comment]
Did you try to manually connect to the minecraft port from the psm hosted system? i.i. trying ``telnet XXX.XXX.XXX.XXX 25510``? What response do you get on that?

**Comment by itsmemac** [comment]
Last error output:
Could not get headers. probably HTTP 50x error.

MySQL db hosted in same websites, gets uptime without issues.
Only minecraft servers shows offline status
Able to ping and get status from other applications

[Screenshot of settings used](https://prnt.sc/uzpjiy)

**Comment by BetTD** [comment]
Please run `telnet <IP address of the server> <port>` as instructed by wienfuchs, ideally from the same machine you're using for PHP Server Monitor.

And also make sure the monitor type on PSM is set to **Service**.
